tests/testthat/test-processes.R

```diff
@@ -73,3 +73,106 @@ test_that("test create_setup_process", {
      delay = c(0, 0)
    )
 })
+
+test_that('create_processes makes a valid process function', {
+
+   S <- individual::State$new("S", 10)
+   human <- individual::Individual$new("human", list(S))
+
+   processes <- create_processes(
+      human$name,
+      c(S$name)
+   )
+
+   for (process in processes) {
+      expect(is.function(process) || inherits(process, 'externalptr'),
+             'Process is not a function')
+   }
+
+})
+
+test_that("test that renderer process works for 1 state", {
```

Comment:
[nit] these tests feel redundant. The individual package tests them [here](https://github.com/mrc-ide/individual/blob/master/tests/testthat/test-render.R#L21). You only really need to test what you implement

Reply:
I think we should keep the second - more than 1 sate - as it we now have a new unction, create_process that wasn't in the code before. Have removed the first test  for renderer with 1 state. We are testing the renderer and the create_process function. Thoughts?

Reply:
yes, we should test create_processes. What was the reason for keeping the second?